### ISSUE TITLE
Feature request - Emergency patch system on stable channel

### ISSUE THREAD

**Issue by MFlyer** [rockstor/rockstor-core#1459]
Hi @schakrava,
while writing you [this](https://github.com/rockstor/rockstor-core/pull/1428#issuecomment-251308270) about Rockstor Widgets and thinking about current chart.js state and last merge about smooth animation (not available on last/our chart.js release), I realized we can suffer same issue over Rockstor. Let me explain:

Assumption: before having a stable channel release we test PRs over testing channel (and we do a lot! :metal:), so everything should be ok, but you know this is not always the real truth.
Our testing channel has 2/3 days average update time, while stable channel has a ~1 month life cycle, so having an unexpected important issue (not minor oddities) could force us to have a short time stable branch update.

My suggestion:
have a new kind of emergency system just using `patch` as well described by @phillxnet [howto](https://forum.rockstor.com/t/how-to-apply-a-patch-to-aid-in-development/1772) on our forum, but all via Rockstor Web UI
IMO you should also mail stable branch user link on patch available (I'm thinking about a copy&paste system to apply patches)

**Comment by sfranzen** [comment]
Wouldn't this be possible within git/GitHub by creating a separate branch, e.g. "stable", from which to tag the stable releases? That way, if some serious regression gets through, you can pull just the bug fix from the master branch and make a new release.

**Comment by phillxnet** [comment]
Closing as no community input for 7 years and we do now have something akin to @sfranzen suggestion re  a "stable" & "testing" GitHub branch arrangement. But have yet to settle into this give the massive changes of late in the "testing" branch. But intend to cherry-pick from fixes in testing to master for hot-fixes pre-tested in testing. This was first done re 4.6.1-0. We can do the like in the future once we have reached the next stable and merged testing into master again.
